doc/getting-started/roscube/roscube-gsg.rst

```diff
@@ -93,18 +93,51 @@ Set Up Environment
    .. code-block:: bash
 
      sudo apt update
-     sudo apt install -y gcc git make gnu-efi libssl-dev libpciaccess-dev \
-       uuid-dev libsystemd-dev libevent-dev libxml2-dev libxml2-utils \
-       libusb-1.0-0-dev python3 python3-pip libblkid-dev \
-       e2fslibs-dev pkg-config libnuma-dev liblz4-tool flex bison
-     sudo pip3 install kconfiglib
+     sudo apt install -y gcc \
+         git \
+         make \
+         vim \
+         libssl-dev \
+         libpciaccess-dev \
+         uuid-dev \
+         libsystemd-dev \
+         libevent-dev \
+         libxml2-dev \
+         libxml2-utils \
+         libusb-1.0-0-dev \
+         python3 \
+         python3-pip \
+         libblkid-dev \
+         e2fslibs-dev \
+         pkg-config \
+         libnuma-dev \
+         liblz4-tool \
+         flex \
+         bison \
+         xsltproc \
+         clang-format \
+         bc
+     sudo pip3 install lxml xmlschema defusedxml
+
+#. Manually fetch and install the ``iasl`` binary to ``/usr/bin`` (where
+   ACRN expects it) with a newer version of the
+   than what's included with Ubuntu 18.04:
```

Comment:
Seems this sentence is not complete. Do you mean "it is newer than the default version installed in Ubuntu 18.04"? Could you please refine it?

Reply:
Thanks for your help, now updated to the latest version of the tool description

Reply:
Thanks for the update. Using the description in latest GSG looks good to me.